config/Symphony.config

```diff
@@ -1,3 +1,5 @@
 {
     "url": "https://foundation-dev.symphony.com"
+    "launchOnStartup": true,
+    "minimizeOnClose": true
```

Comment:
As the code to load/save minimizeOnClose settings is almost the same of auto launch, I have already prepared it together so ELECTRON-40 has only to implement the menu behavior itself.